### PR TITLE
Gear colors are now retained on restart

### PR DESCRIPTION
**Pull request by Scar26**
Fixes : #336

### CODE REVIEW

File: activities/Gears.activity/lib/gearsketch/gearsketch_main.js [hunk 1]
```diff
@@ -447,6 +447,7 @@
       gearCanvas.width = size;
       ctx = gearCanvas.getContext("2d");
       gearCopy = new Gear(new Point(0.5 * size, 0.5 * size), 0, gear.numberOfTeeth, gear.id);
```
Comment (NikhilM98) [inline activities/Gears.activity/lib/gearsketch/gearsketch_main.js]: Call `new Gear` with `gear.hue`

Comment (Scar26) [reply activities/Gears.activity/lib/gearsketch/gearsketch_main.js]: Done

**Comment by Scar26** [comment]
@llaske Can you please review this PR

**Comment by Scar26** [comment]
@llaske can you please review this PR?

**Comment by llaske** [comment]
Good job.

**Pull request merged**